nrf_dm/Kconfig

```
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NRF_DM
+	bool "nRF DM (Distance Measurement) library [EXPERIMENTAL]"
+	select FPU
```

Maybe?
```suggestion
	select FPU if CPU_HAS_FPU
```
To consider an SoC without an FPU